components/TravelDetails/index.js

```diff
@@ -1,37 +1,44 @@
 import { travel, user } from "@/data/data";
-import Image from "next/image";
 import { styled } from "styled-components";
+import UserIcon from "../UserIcon";
+
+const trip = travel[0];
 
 export default function TravelOverview() {
   return (
-    <article>
-      {travel.map((trip) => {
-        return (
-          <>
-            <section>
-              {trip.name}
-              <br />
-              📍{trip.destination}
-            </section>
-
-            <section>
-              {UserIcon()}
-              <p>Time-period: {trip.timePeriod}</p>
-            </section>
-          </>
-        );
-      })}
-    </article>
+    <FlexContainer>
+      <article>
+        {trip.name}
+        <br />
+        📍{trip.destination}
+      </article>
+      <FlexArticle>
+        <UserIcon />
```

Comment:
In your US you mentioned - "The left box contains a person-icon and the names of the members, who join the respective trip" - please add the name of the Person.